qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -143,19 +147,11 @@ def _vf2_match_not_found(property_set):
         _improve_layout = NoiseAdaptiveLayout(backend_properties)
     elif layout_method == "sabre":
         _improve_layout = SabreLayout(coupling_map, max_iterations=2, seed=seed_transpiler)
-    else:
-        raise TranspilerError("Invalid layout method %s." % layout_method)
 
     toqm_pass = False
-    if routing_method == "basic":
-        routing_pass = BasicSwap(coupling_map)
-    elif routing_method == "stochastic":
-        routing_pass = StochasticSwap(coupling_map, trials=20, seed=seed_transpiler)
-    elif routing_method == "lookahead":
-        routing_pass = LookaheadSwap(coupling_map, search_depth=4, search_width=4)
-    elif routing_method == "sabre":
-        routing_pass = SabreSwap(coupling_map, heuristic="lookahead", seed=seed_transpiler)
-    elif routing_method == "toqm":
+    routing_pm = None
+    # TODO: Remove when qiskit-toqm has it's own plugin and we can rely on just the plugin interface
+    if routing_method == "toqm":
```

Comment:
not yet, because it's a qiskit-toqm and depends on this PR being released before it is actionable. (qiskit-toqm can't use this plugin interface until it's available in a release).